Resources/public/css/theme/views/fields/edit/country.css

```diff
@@ -0,0 +1,10 @@
+.ez-view-countryeditview .ez-field-description {
```

Comment:
that's a lot of duplicated styles, you should add class on the field edit view container that triggers the default field description.